In [ ]:
from pathlib import Path
import pandas as pd

In [ ]:
MAIN_DIR = Path("/home/users/lumori/bbtautau/")
for c in ["muon", "hadronic", "electron"]:
    plot_dir = MAIN_DIR / f"plots/SensitivityStudy/25Mar7{c}"
    df = pd.read_csv(plot_dir / "2022_2022EE_2023_2023BPix-results_fast.csv")
    df = df.rename(columns={"Unnamed: 0": c})
    df = df.drop(index=3)
    print(df.to_markdown(index=False))
    print("\n")

| muon               |       B=1 |       B=2 |       B=8 |   Max_significance |
|:-------------------|----------:|----------:|----------:|-------------------:|
| Sig_Yield          |  0.137415 |  0.160491 |  0.185217 |           0.137415 |
| BG_Yield           |  1        |  2        |  8        |           1        |
| Limit              | 14.5545   | 17.6236   | 30.5417   |          14.5545   |
| Limit_scaled_22_24 |  4.83601  |  5.85578  | 10.1481   |           4.83601  |
| Limit_scaled_Run3  |  3.47058  |  4.20242  |  7.2828   |           3.47058  |


| hadronic           |       B=1 |         B=2 |      B=8 |   Max_significance |
|:-------------------|----------:|------------:|---------:|-------------------:|
| Sig_Yield          |  0.026628 |   0.0278628 |  0.12528 |          0.0937288 |
| BG_Yield           |  1        |   2         |  8       |          4         |
| Limit              | 75.1088   | 101.513     | 45.1536  |         42.6763    |
| Limit_scaled_22_24 | 24.9564   

| muon               |       B=1 |       B=2 |       B=8 |   Max_significance |
|:-------------------|----------:|----------:|----------:|-------------------:|
| Sig_Yield          |  0.137415 |  0.160491 |  0.185217 |           0.137415 |
| BG_Yield           |  1        |  2        |  8        |           1        |
| Limit              | 14.5545   | 17.6236   | 30.5417   |          14.5545   |
| Limit_scaled_22_24 |  4.83601  |  5.85578  | 10.1481   |           4.83601  |
| Limit_scaled_Run3  |  3.47058  |  4.20242  |  7.2828   |           3.47058  |



| hadronic           |       B=1 |         B=2 |      B=8 |   Max_significance |
|:-------------------|----------:|------------:|---------:|-------------------:|
| Sig_Yield          |  0.026628 |   0.0278628 |  0.12528 |          0.0937288 |
| BG_Yield           |  1        |   2         |  8       |          4         |
| Limit              | 75.1088   | 101.513     | 45.1536  |         42.6763    |
| Limit_scaled_22_24 | 24.9564   |  33.7296    | 15.0032  |         14.1801    |
| Limit_scaled_Run3  | 17.91     |  24.2061    | 10.7671  |         10.1764    |



| electron           |        B=1 |        B=2 |       B=8 |   Max_significance |
|:-------------------|-----------:|-----------:|----------:|-------------------:|
| Sig_Yield          |  0.0240733 |  0.0410527 |  0.111333 |           0.148758 |
| BG_Yield           |  1         |  2         |  8        |          12        |
| Limit              | 83.0798    | 68.8974    | 50.8104   |          46.5738   |
| Limit_scaled_22_24 | 27.6049    | 22.8926    | 16.8828   |          15.4751   |
| Limit_scaled_Run3  | 19.8107    | 16.4289    | 12.116    |          11.1057   |



| Year           |   Lumi [fb^-1] |       Datasets |      Eras |   
|:-------------------|-----------:|-----------:|----------:|
| 2022          |  8.0 |  0.0410527 |  |    
| 2022EE           |  26.7        |  2         |  8        |  
| 2023              | 17.8     | 68.8974    | 50.8104   |    
| 2023BPix | 9.5   | 22.8926    | 16.8828   |    